bootstrap/cmd/bootstrap/app/ksServer.go

```diff
@@ -63,7 +65,7 @@ type ksServer struct {
 
 	fs afero.Fs
 
-	apps      map[string]*appInfo
+	projectLocks map[string]*sync.Mutex
```

Comment:
I guess the map key is the project name and the mutex for each project? Could you add comment.